Sources/Core/Views/ProfileImageView/ProfileImageView.swift

```diff
@@ -22,8 +22,8 @@ public class ProfileImageProperties: ObservableObject {
 
 @available(iOS 14, *)
 public struct ProfileImageView: View {
-  public var imageTapped = Delegated<Void, Void>()
-  public var badgeTapped = Delegated<Void, Void>()
```

Comment:
What happened to `Delegated`?

Reply:
I reverted to closures b/c `Delegated` pattern doesn't work with `struct` (SwiftUI Views are structs).

Reply:
Good point; value types do not cause reference cycles so regular closures should be okay.